standards/nist_secure-software-development-framework_1.1.json

```diff
@@ -0,0 +1,562 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "serialNumber": "urn:uuid:611dd916-ef79-410d-8188-417686a6b968",
+  "version": 1,
+  "metadata": {
+    "manufacture": {
+      "name": "OWASP Foundation",
+      "url": [
+        "https://owasp.org/",
+        "https://cyclonedx.org/",
+        "https://github.com/CycloneDX/official-3rd-party-standards"
+      ]
+    }
+  },
+  "declarations": {
+    "standards": [
+      {
+        "bom-ref": "ssdf-1.1",
+        "name": "Secure Software Development Framework (SSDF) Version 1.1",
+        "description": "NIST Special Publication 800-218",
+        "version": "1.1",
+        "owner": "National Institute of Standards and Technology",
+        "requirements": [
+
+          {
+            "bom-ref": "ssdf-1.1-PO.1",
+            "identifier": "PO.1",
+            "text": "Ensure that security requirements for software development are known at all times so that they can be taken into account throughout the SDLC and duplication of effort can be minimized because the requirements information can be collected once and shared. This includes requirements from internal sources (e.g., the organization’s policies, business objectives, and risk management strategy) and external sources (e.g., applicable laws and regulations).",
+            "opencre": null,
+            "parent": "",
```

Comment:
On another note: Should we have a root node for the standard or is that just implied?

Reply:
See https://github.com/CycloneDX/official-3rd-party-standards/pull/1#discussion_r1260178227